utility/src/main/java/org/oppia/android/util/parser/BlurTransformation.kt

```diff
@@ -0,0 +1,65 @@
+package org.oppia.android.util.parser
+
+import android.content.Context
+import android.graphics.Bitmap
+import android.renderscript.Allocation
+import android.renderscript.Element
+import android.renderscript.RenderScript
+import android.renderscript.ScriptIntrinsicBlur
+import com.bumptech.glide.load.engine.bitmap_recycle.BitmapPool
+import com.bumptech.glide.load.resource.bitmap.BitmapTransformation
+import java.security.MessageDigest
+
+// The radius of the blur, a float between 0 and 25. It defines the value of the standard
+// deviation to the Gaussian function.
+private const val BLUR_RADIUS = 20f
+
+/**
+ * [BlurTransformation] is a bitmap transformation that blurs an image using RenderScript.
+ *
+ * The following [tutorial](https://futurestud.io/tutorials/glide-custom-transformation)
+ * was used as a reference, as well as this [article](https://stackoverflow.com/a/23119957).
+ * */
+
+class BlurTransformation(private val context: Context) : BitmapTransformation() {
+
+  private val renderScript by lazy { RenderScript.create(context) }
+
+  override fun transform(
+    pool: BitmapPool,
+    toTransform: Bitmap,
+    outWidth: Int,
+    outHeight: Int
+  ): Bitmap {
+
+    val inputAllocation = if (toTransform.config == Bitmap.Config.ARGB_8888) {
+      // Create a RenderScript allocation pointing to the original bitmap to avoid a memory copy.
+      Allocation.createFromBitmap(renderScript, toTransform)
+    } else {
+      // Create a RenderScript allocation pointing to the a copy.
```

Comment:
```suggestion
      // Create a RenderScript allocation pointing to a copy. Note that a copy is needed here since the original bitmap configuration isn't in ARGB_8888 (which is required for how the script intrinsic is configured below).
```

(needs to be broke into multiple lines) but this should help clarify the need a bit. We could alternatively try selecting a different element type below, but I think the copy is reasonable.